eel/tests/persistence_test.rs

```diff
@@ -57,7 +67,27 @@ mod persistence_test {
             .contains("RemoteStorageServiceUnavailable"));
     }
 
-    fn run_flow(node_handle: &NodeHandle) {
+    fn run_flow_normal_restart(node_handle: &NodeHandle) {
+        run_flow_1(node_handle);
+
+        // Wait for shutdown to complete
```

Comment:
`// Wait for eel-node to shutdown`

Also for the same comment in other places. Just want to be explicit about what exactly is shutting down.

Reply:
[8c9a5a3](https://github.com/getlipa/lipa-lightning-lib/pull/204/commits/8c9a5a374e16bdffeb6aa3dbe9255fb8af275c8e)